warehouse/templates/manage/manage_base.html

```diff
@@ -204,6 +204,14 @@ <h3 class="sidebar-section__title">{% trans %}Your account{% endtrans %}</h3>
                   {% trans %}Your projects{% endtrans %}
                 </a>
               </li>
+              {% if not request.flags.enabled(AdminFlagValue.DISABLE_ORGANIZATIONS) %}
```

Comment:
Just a smell on the double negative. I think It's OK as is... but 

`{% if request.flags.enabled(AdminFlagValue.ENABLE_ORGANIZATIONS) %}` is just screaming at me through this string.

Reply:
Not worth addressing at this time. Just a note.